CHANGELOG.md

```diff
@@ -11,11 +11,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## Added
 
 - Added `Marshler` config option to `otlphttp` to enable otlp over json or protobufs. (#1586)
+
 ### Removed
 
 - Removed the exported `SimpleSpanProcessor` and `BatchSpanProcessor` structs.
    These are now returned as a SpanProcessor interface from their respective constructors. (#1638)
-
+- Removed setting `ErrorStatus`  while recording err as span event in `RecordError`. (#1663)
```

Comment:
This would fit better in a `Changed` section as that would better fit the perspective of the user of this code.

```suggestion
- Removed setting status to `Error` while recording an error as a span event in `RecordError`. (#1663)

```

Reply:
I think it can fit in both Changed and Removed. I have placed it because behavior of code changed and code removed.